src/SourceGenerators/Uno.UI.SourceGenerators/Content/Uno.UI.SourceGenerators.props

```diff
@@ -393,7 +393,7 @@
 
 	<Target Name="Roslyn51768Workaround"
 			BeforeTargets="BeforeBuild"
-					Condition="'$(DisableRoslyn51768Workaround)'==''">
+					Condition="'$(DisableRoslyn51768Workaround)'=='' and '$(DesignTimeBuild)'=='true'">
```

Comment:
This didn't work. The generated editorconfig is still seen as newer than output file. This might be a bug on Roslyn side. Reverting this for now.